girderformindlogger/api/v1/user.py

```diff
@@ -812,19 +812,21 @@ def getUsersDetails(self):
         .errorResponse(('You do not have write access for this user.',
                         'Must be an admin to create an admin.'), 403)
     )
+
     def updateUser(
         self,
         user,
+        login=None,
         displayName="",
         email="",
         admin=False,
         status=None,
-        firstName=None,
-        lastName=None
+        firstName=None
     ): # 🔥 delete firstName and lastName once fully deprecated
-        user['firstName'] = displayName if len(
-            displayName
-        ) else firstName if firstName is not None else ""
+
+        login = cherrypy.request.body.params['login']
```

Comment:
If we're adding a param to a call, we should in its decorator rather than picking it out of the request like this.